tests/integration/targets/copr/aliases

```diff
@@ -0,0 +1,2 @@
+needs/root
```

Comment:
The most important line is missing:
```
shippable/posix/group1
```
Otherwise the tests are not run on any target.